packages/vm/README.md

```diff
@@ -7,7 +7,7 @@
 [![Discord][discord-badge]][discord-link]
 
 | TypeScript implementation of the Ethereum VM. |
-| --- |
+| --------------------------------------------- |
```

Comment:
Prettier linter did this, should I remove it?

Reply:
It's fine you can leave it, as it will be fixed moving forward 👍